src/libraries/System.Runtime/tests/System/HashCodeTests.cs

```diff
@@ -121,6 +122,75 @@ public static void HashCode_Add_NullEqualityComparer()
         Assert.Equal(expected.ToHashCode(), hc.ToHashCode());
     }
 
+    [Fact]
+    public static void HashCode_Add_Span_AllValuesHashed()
+    {
+        static void Populate(ref HashCode hc, int length)
+        {
+            byte[] data = Enumerable.Range(1, length).Select(i => (byte)i).ToArray();
+            hc.Add((ReadOnlySpan<byte>)data);
```

Comment:
Is the case necessary to disambiguate from the generic method overload? Should we have named this something different? 🙀

Reply:
Without it, overload resolution has this calling `hc.Add<byte[]>(data)` which is... unexpected :)

Reply:
(We could also add a byte[] overload, which would fix that.)

Reply:
I think we should either add a `byte[]` overload or name the method differently.

Reply:
Per offline discussions, changed it to be AddBytes.